app/controllers/lists_controller.rb

```diff
@@ -48,4 +51,8 @@ def destroy_multiple
   def list_params
     params.require(:list).permit(:id, :therapist_id, :title, :description)
   end
+
+  def require_owned_wishlist!
+    current_user.role.lists.map{|list| list.id}.include?(params[:id])
```

Comment:
if you're trying to get an attribute from a model, you can just do `lists.pluck(:id)`, which gets it through a SQL select statement instead of loading the entire set of objects into memory and mapping through it to get one attribute